src/OpenLoco/src/Graphics/SoftwareDrawingEngine.h

```diff
@@ -72,7 +72,7 @@ namespace OpenLoco::Gfx
 
         SDL_Texture* _screenRGBATexture{};
 
-        std::unique_ptr<SoftwareDrawingContext> _ctx;
+        SoftwareDrawingContext _ctx;
```

Comment:
Could you elaborate on this change?

Reply:
It returns the drawing context by reference and resize changed the pointer so that was bad. Now the drawing context remains valid between resize, storing an old RenderTarget isn't but one shouldn't do that anyway.